<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>895</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>895</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>895</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>541</width>
     <height>901</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#frame{background-color: #f7f7f7ff;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66,73,84);
border-bottom-color: lineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(246, 175, 63, 255));

font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Registration Form</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>250</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 148, 26)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66,73,84);
font: 63 11pt &quot;Open Sans Semibold&quot;;
</string>
    </property>
    <property name="text">
     <string>Please Fill In Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regbtn_sub">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>800</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 148, 26);
color: #f7f7f7ff;
font: 10pt Open Sans Semibold;</string>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>325</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 63 28pt &quot;Open Sans Semibold&quot;;
color: rgb(255,148,26)</string>
       </property>
       <property name="text">
        <string>Ohio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(66,73,84);
font: 63 28pt &quot;Open Sans Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Academy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>271</y>
      <width>94</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66, 73, 84);
font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname_in">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>271</y>
      <width>137</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastname_in">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>321</y>
      <width>137</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>321</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
color: rgb(66, 73, 84)</string>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="middlename_in">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>371</y>
      <width>123</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>371</y>
      <width>116</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66, 73, 84);
font: 63 10pt &quot;Open Sans Semibold&quot;;
</string>
    </property>
    <property name="text">
     <string>Middle Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>421</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
color: rgb(63, 73, 84);</string>
    </property>
    <property name="text">
     <string>Sex:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_radio">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>421</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(63, 73, 84);

font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="male_radio">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>423</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(66, 73, 84);
color:rgb(66, 73, 84);
font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>471</y>
      <width>66</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
color: rgb(66, 73, 84);</string>
    </property>
    <property name="text">
     <string>Email:    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_in">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>471</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="department_in">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>521</y>
      <width>117</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>521</y>
      <width>126</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
color: rgb(66, 73, 84);
</string>
    </property>
    <property name="text">
     <string>Department:     </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>571</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66, 73, 84);
font: 10pt Open Sans Semibold;</string>
    </property>
    <property name="text">
     <string>Academic Session:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schoolsession_in">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>571</y>
      <width>137</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>621</y>
      <width>68</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(66, 73, 84);

font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Level:     </string>
    </property>
   </widget>
   <widget class="QComboBox" name="selection_combo">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>621</y>
      <width>106</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt Open Sans Semibold;
color: rgb(66, 73, 84);
</string>
    </property>
    <item>
     <property name="text">
      <string>100 Level</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 Level</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 Level</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>400 Level</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500 Level</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>681</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt Open Sans Semibold;
color: rgb(66, 73, 84);</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_in">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>681</y>
      <width>137</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>731</y>
      <width>153</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
color: rgb(66, 73, 84);</string>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmpass_in">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>731</y>
      <width>137</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
